docs/pages/database-access/guides/gui-clients.mdx

```diff
@@ -353,3 +353,75 @@ Click `Add Redis Database`.
 Congratulations! You have just connected to your Redis instance.
 
 ![Redis Insight Connected](../../../img/database-access/guides/redis/redisinsight-connected.png)
+
+
+## Snowflake - JetBrains (IntelliJ, Goland, DataGrip, PyCharm etc.)
+
+<Admonition type="note">
```

Comment:
If we are expecting all readers to run this command, I think we should remove the `Admonition` tags and frame this instruction as the first step in this section. Right now, it looks like an aside, and a user might bypass this code snippet without running the proxy.